pillar/heroku/mitxonline.sls

```diff
@@ -0,0 +1,119 @@
+    AWS_ACCESS_KEY_ID:  __vault__:cache:aws-mitx/creds/read-write-delete-xpro-app-{{ env_data.env_name }}>data>access_key
+    AWS_SECRET_ACCESS_KEY: __vault__:cache:aws-mitx/creds/read-write-delete-xpro-app-{{ env_data.env_name }}>data>secret_key
+    AWS_STORAGE_BUCKET_NAME: 'xpro-app-{{ env_data.env_name }}'
+    {% if env_data.env_name == 'production' %}
+    {% set pg_creds = salt.vault.cached_read('postgres-production-apps-mitxonline/creds/mitxonline', cache_prefix='heroku-mitxonline') %}
+    DATABASE_URL: postgres://{{ pg_creds.data.username }}:{{ pg_creds.data.password }}@{{ rds_endpoint }}/mitxonline
+    HIREFIRE_TOKEN: __vault__::secret-{{ business_unit }}/production-apps/hirefire_token>data>value
+    {% endif %}
+    GA_TRACKING_ID: {{ env_data.GOOGLE_TRACKING_ID }}
+    GTM_TRACKING_ID: {{ env_data.GOOGLE_TAG_MANAGER_ID }}
+    MAILGUN_KEY: __vault__::secret-operations/global/mailgun-api-key>data>value
+    MAILGUN_FROM_EMAIL: {{ env_data.MAILGUN_FROM_EMAIL }}
+    MAILGUN_SENDER_DOMAIN: {{ env_data.MAILGUN_SENDER_DOMAIN }}
+    MITXONLINE_ADMIN_EMAIL: 'cuddle-bunnies@mit.edu'
+    MITXONLONE_BASE_URL: {{ env_data.MITXONLINE_BASE_URL }}
+    MITXONLONE_DB_CONN_MAX_AGE: 0
+    MITXONLNE_DB_DISABLE_SSL: True    # pgbouncer buildpack uses stunnel to handle encryption
+    MITXONLINE_ENVIRONMENT: {{ env_data.env_name }}
+    MITXONLINE_FROM_EMAIL: 'MITx Online <support@mitxonline.mit.edu>'
+    MITXONLINE_LOG_LEVEL: {{ env_data.app_log_level }}
+    MITXONLINE_OAUTH_PROVIDER: 'mitxonline-oauth2'
+    MITXONLINE_REGISTRATION_ACCESS_TOKEN:  __vault__:gen_if_missing:secret-{{ business_unit }}/{{ env_data.openedx_environment }}/mitxonline-registration-access-token>data>value
+    MITXONLINE_REPLY_TO_ADDRESS: 'MITx Online <support@mitxonline.mit.edu>'
+    MITXONLINE_SECURE_SSL_REDIRECT: True
+    MITXONLINE_SECURE_SSL_HOST: {{ env_data.MITXONLINE_SECURE_SSL_HOST }}
+    MITXONLINE_SUPPORT_EMAIL: {{ smtp_config.data.support_email }}
+    MITXONLINE_USE_S3: True
+    NODE_MODULES_CACHE: False
+    OPENEDX_API_BASE_URL: {{ env_data.OPENEDX_API_BASE_URL}}
+    OPENEDX_API_CLIENT_ID: __vault__::secret-{{ business_unit }}/{{ environment }}/openedx-api-client>data>client-id
+    OPENEDX_API_CLIENT_SECRET: __vault__::secret-{{ business_unit }}/{{ environment }}/openedx-api-client>data>client-secret
+    OPENEDX_API_KEY: __vault__:gen_if_missing:secret-{{ business_unit }}/{{ env_data.openedx_environment }}/edx-api-key>data>value
+    # This can be removed once PR#1314 is in production
+    OPENEDX_GRADES_API_TOKEN:  __vault__::secret-{{ business_unit }}/{{ environment }}/openedx-grades-api-token>data>value
+    OPENEDX_OAUTH_APP_NAME: 'edx-oauth-app'
+    # This replaces OPENEDX_GRADES_API_TOKEN and is tied to xpro-grades-api user in openedx
+    OPENEDX_SERVICE_WORKER_API_TOKEN: __vault__::secret-{{ business_unit }}/{{ environment }}/openedx-service-worker-api-token>data>value
+    OPENEDX_SERVICE_WORKER_USERNAME: mitxonline-service-worker-api
```

Comment:
Does this comment mean that OPENEDX_GRADES_API_TOKEN is obsolete?

Reply:
I believe so yeah